sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -344,6 +344,14 @@ final class InsertIntoHadoopFsRelationCommandMeta(
 
   private var fileFormat: Option[ColumnarFileFormat] = None
 
+  override def checkTimeZone(): Boolean =
+    if (GpuOrcFileFormat.isSparkOrcFormat(cmd.fileFormat.getClass)) {
+      // ORC always uses UTC when writing timestamps, it's irrelevant to timezone.
```

Comment:
Yes and no. ORC has support for other time zones to show up in an ORC file. https://orc.apache.org/specification/ORCv1/ in the section for StripeFooter. I am fine with us assuming that it is not there for the purpose of deciding what to place on the GPU and what not to, but I am not okay with us saying that it cannot happen.